build/android/lzma_compress.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2015 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+# pylint: disable=F0401
+
+import optparse
+import os
+import shutil
+import sys
+import subprocess
+
+GYP_ANDROID_DIR = os.path.join(os.path.dirname(__file__),
+                               os.pardir, os.pardir, os.pardir,
+                               'build',
+                               'android',
+                               'gyp')
+sys.path.append(GYP_ANDROID_DIR)
+
+from util import build_utils
+
+def main():
+  parser = optparse.OptionParser()
+  parser.add_option('--dest-path', help='Destination directory for compressed files')
+  parser.add_option('--sources', help='The list of files to be comprossed')
+  parser.add_option('--stamp', help='The file to touch on success.')
+
+  options, _ = parser.parse_args(sys.argv)
+
+  if not os.path.exists(options.dest_path):
```

Comment:
You could use `build_utils.MakeDirectory()` and avoid having to check for the path.